contrib/mmap/src/IntermediateValues.cpp

```diff
@@ -254,7 +254,7 @@ namespace MMAP
         fwrite(&b, sizeof(char), 1, objFile);
         fclose(objFile);
 
-        sprintf(objFileName, "meshes/%03u%02u%02u.mesh", mapID, tileY, tileX);
+        sprintf(objFileName, "%s/meshes/%03u%02u%02u.mesh", mapID, tileY, tileX);
```

Comment:
You're lacking a string argument here, so this call is not going to end well.

Reply:
yeah - saw that too already fixed locally ;)